build.gradle

```diff
@@ -2,7 +2,19 @@
 apply plugin: 'maven-publish'
 
 group = 'org.elasticsearch'
-version = '4.4.0'
+String upstreamVersion = '4.4.0'
+
+// Rarely, modifications may need to be made to the jna jar that
+// aren't reflected in the upstream.  A suffix can be supplied to bump
+// the local version.  See readme for details.
+String versionSuffix = System.getProperty("version.suffix")
+
+if (versionSuffix != null) {
```

Comment:
You can avoid this conditional by using a default value of "" in the getProperty call above.